frontend/packages/ux-editor/src/components/UnsupportedVersionMessage/UnsupportedVersionMessage.module.css

```
@@ -0,0 +1,15 @@
+.wrapper {
+  display: flex;
+  height: calc(100vh - var(--header-height));
+  align-items: center;
+  justify-content: center;
+}
+
+.message {
+  width: 600px;
```

By using max-width and width combined like this, it would also be responsive alert. 😊 
```suggestion
  max-width: 600px;
  width: 100%;
```

---

Did the suggestion above not work? 🤔

---

It did - I had some issues with merging, must have gotten lost 🤔 I pushed a new commit with the changes now.